setuptools/config.py

```diff
@@ -245,8 +245,8 @@ def _parse_file(cls, value):
         directory with setup.py.
 
         Examples:
-            include: LICENSE
-            include: src/file.txt
+            file: LICENSE
```

Comment:
I presume this is a correction and not a change.

Reply:
yes